File: runners/nodejs/runner.js
function runFunctionWithArgs(jsonArgs) {
  try {
    const args = JSON.parse(jsonArgs);

    if (!Array.isArray(args)) {
      throw new Error("Arguments must be an array.");
    }

    const result = main(...args);
    console.log(JSON.stringify(result));
    process.exit(0);
  } catch (e) {
    console.error(JSON.stringify(e.stack || e));
    process.exit(1);
  }
}

const jsonArgs = process.argv[2];

runFunctionWithArgs(jsonArgs);

File: runners/python/runner.py
import json
import sys
import traceback


def run_function_with_args(json_args):
    try:
        args = json.loads(json_args)

        if not isinstance(args, list):
            raise ValueError("Arguments must be an array.")

        result = main(*args)
        print(json.dumps(result))
        sys.exit(0)
    except Exception as e:
        print(json.dumps("\n".join(traceback.format_exception(e))), file=sys.stderr)
        sys.exit(1)


if __name__ == "__main__":
    if len(sys.argv) > 1:
        json_args = sys.argv[1]
        run_function_with_args(json_args)
    else:
        print("No arguments provided", file=sys.stderr)
        sys.exit(1)
